task/docker-build/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Add an internal registry as sidecar to the task so we can upload it directly
+# from our tests without having to go to an external registry.
+add_sidecar_registry ${TMPF}
+
+# Add git-clone
+kubectl -n ${tns} apply -f ../../../git-clone/0.1/git-clone.yaml
```

Comment:
Sorry for the changes requested, there's a current PR in progress to fix the CI : https://github.com/tektoncd/catalog/pull/421

I think this might change the path here as well else while the test would fail

For e.g. 

https://github.com/tektoncd/catalog/blob/6e3d28be12825d03193113d886d79d5e9175c922/task/buildah/0.1/tests/pre-apply-task-hook.sh#L8

Can you please update this changes

Reply:
@PuneetPunamiya :+1: